YoseptF/my-eat-list-backend
# Feature/setup

# My Eat List Backed

## The information source for [My eat list](https://github.com/YoseptF/my-eat-list-frontend)

![header](https://i.imgur.com/ZON2pOE.png)

---

With this app, you can save a daily list of what you eat from a list and check your recommended calories based on height, age, and weight.
![relationships](https://i.imgur.com/UWwEHi9.png)

---

# Development Setup

Steps: 
- Clone this repo:
```
$ git clone git@github.com:YoseptF/my-eat-list-backend.git
```
- Move to the lastest branch:
```
$ git checkout [branch name]
```
- Install the dependencies:
```
$ yarn install --check-files
```
```
$ bundle
```
- Migrate and Seed:
```
$ bundle db:migrate db:seed
```
- Run the developer server: (this one is gonna be pretty important if you are running both front and backend, because by default rails and react try to use `localhost:3000`, so I configured the npm command to run this on `localhost:4000`)
```
$ yarn start
```
